src/services/control.js

```diff
@@ -305,49 +327,62 @@ var gitPull = function (repoPath, callback) {
 // Read functions
 
 var readContentMap = function (callback) {
-  var contentMapPath = PathService.getConfigPath(config.control_content_file());
+  var contentFiles = globby.sync([
+    PathService.getConfigPath(config.control_content_file()),
+    PathService.getConfigPath('content.d/**/*')
+  ]);
```

Comment:
Putting together these globs is probably best done in PathService anyways. I think using the directory only if a config file hasn't been explicitly provided via the environment variable makes sense. Are those environment variables being explicitly set when we run the presenter in production?